css-four-column-layout/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>CSS Challenge - Four Column Layout</title>
+    <link href="https://fonts.googleapis.com/css2?family=Barlow:wght@400;700&display=swap" rel="stylesheet">
+    <link href="styles.css" rel="stylesheet">
+  </head>
+  <body>
+    <div class="container display-container">
+      <div class="row">
+        <div class="column-full">
+          <h1>How It Works</h1>
+        </div>
+      </div>
+      <div class="row display-row">
+        <div class="column-one-fifth">
+          <img class="screen" src="images/screen-1.png" alt="app switching screen">
+          <img class="icon" src="images/switch-icon.png" alt="switching icon">
+          <h2>App Switching</h2>
+          <p>Toggles all your apps on and off.</p>
+        </div>
+        <div class="column-one-fifth">
+          <img class="screen" src="images/screen-2.png" alt="trip filtering screen">
+          <img class="icon" src="images/filter-icon.png" alt="filtering icon">
+          <h2>Trip Filtering</h2>
+          <p>Evaluates your trip requests.</p>
+        </div>
+        <div class="column-one-fifth">
+          <img class="screen" src="images/screen-3.png" alt="service filters screen">
+          <img class="icon" src="images/slider-icon.png" alt="slider icon">
+          <h2>Service Filters</h2>
+          <p>Accepts only your ideal trips.</p>
+        </div>
+        <div class="column-one-fifth">
+          <img class="screen" src="images/screen-4.png" alt="stats screen">
+          <img class="icon" src="images/bars-icon.png" alt="bars icon">
+          <h2>Stats + Mileage</h2>
+          <p>Tracks your trips across all apps.</p>
+        </div>
```

Comment:
Instead of using `column-fifth` to shrink the content, I would stick to using `column-fourth` and just add some `padding` on the `row` to shrink how much space it has available.